projects/packages/assets/src/class-script-data.php

```diff
@@ -86,17 +87,18 @@ public static function render_script_data() {
 		self::$did_render_script_data = true;
 
 		$script_data = is_admin() ? self::get_admin_script_data() : self::get_public_script_data();
-
-		$script_data = wp_json_encode(
+		$script_json = wp_json_encode(
```

Comment:
Why did you change `$script_data` to `$script_json` rather than keeping it as it was?

Reply:
The name change wasn't needed, I just chose to prevent the variable reassignment and add clarity with what data the variable holds, but it does create a new variable. I'm happy to switch back to `$script_data` here.